content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -105,6 +105,32 @@ in your **Hosted endpoints** and insert the following data into the pop-up form:
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+Configuration tab
+-----------------
+
+.. _stripe/manual_capture:
+
+Place a hold on a card
+~~~~~~~~~~~~~~~~~~~~~~
+
+Stripe allows you to **capture an amount manually** instead of having an immediate capture. This
+feature earmarks an amount on the customer's account for a determined period of time that depends on
+the payment method used.
+
+To set it up, enable the :guilabel:`Capture Amount Manually` option on Odoo, as explained in the
+:ref:`payment acquirers documentation <payment_acquirers/capture_amount>`.
+
+.. warning::
+   When the manual capture feature is activated, your customers won't be able to proceed with any
+   payment if unsupported payment methods are selected.
```

Comment:
If you want to write future tense, then I'd replace "when" with "Once the manual capture, ..."
Otherwise, I'd rather go with the present tense since it's normal expected behavior.
```suggestion
   When the manual capture feature is activated, your customers are not able to proceed with any
   payment if unsupported payment methods are selected.
```